docs/basic-features/pages.md

```diff
@@ -86,7 +86,7 @@ function About() {
 export default About
 ```
 
-Note that this page does not need to fetch any external data to be pre-rendered. In cases like this, Next.js generates a single HTML file per page during build time.
+**Note**: that this page does not need to fetch any external data to be pre-rendered. In cases like this, Next.js generates a single HTML file per page during build time.
```

Comment:
All the "Note that" should be reverted

Reply:
https://github.com/vercel/next.js/pull/48417/commits/7aab58d49c0d238aaebfd22d407f411d15009b08 Reverted the change as suggested, since it's a continuation, not a callout.